db/migrate/20160224175308_create_tools.rb

```diff
@@ -0,0 +1,11 @@
+class CreateTools < ActiveRecord::Migration
+  def change
+    create_table :tools do |t|
+      t.string :title
+      t.text :description
+      t.integer :rents, default: 0
```

Comment:
I know! I'm meaning to implement a renting model aswell and the rent attribute in tool is currently just for future use.